ops/provision.sh

```diff
@@ -1,35 +1,32 @@
-echo "------------------------------------------------------------------"
-echo "------------------ Provision package lists -----------------------" 
-echo "------------------------------------------------------------------"
+echo "------ Provision package lists ------"
 
 sudo apt-get -y update
+sudo apt-get -y unzip
 
-echo "------------------------------------------------------------------"
-echo "--------------------- Install ruby -------------------------------" 
-echo "------------------------------------------------------------------"
+echo "------ Install ruby ------"
 
 curl -L https://get.rvm.io | bash -s stable --ruby=2.2.1-dev
 
 sudo apt-get install -y ruby-all-dev
 
-echo "------------------------------------------------------------------"
-echo "--------------------- Install gems -------------------------------" 
-echo "------------------------------------------------------------------"
+echo "------ Install bundler ------"
 
+sudo gem install bundler
+
+echo "------ Install sqlite3 ------"
 
-sudo gem install activerecord
-sudo gem install octokit
 sudo apt-get install libsqlite3-dev
 sudo apt-get install sqlite3
-sudo gem install sqlite3
-sudo gem install json
-sudo gem install sinatra
-sudo gem install haml
-sudo gem install time_difference
-
-echo "------------------------------------------------------------------"
-echo "--------------------- Add cron jobs ------------------------------" 
-echo "------------------------------------------------------------------"
-
-echo "0 * * * * /home/ubuntu/daily_report.sh
-*/15 * * * * /home/ubuntu/conflict_checker.sh" | crontab -
+
+echo "------ Install gems ------"
+
+sudo bundle install
```

Comment:
did you check it is actually working?
A Gemfile should be immediately available to Bundler to pick it up.
http://bundler.io/v1.11/bundle_install.html